pkg/helm/run.go

```diff
@@ -49,12 +50,18 @@ func Run(flags *hoflags.HelmOperatorFlags) {
 
 	printVersion()
 
-	namespace, found := os.LookupEnv(k8sutil.WatchNamespaceEnvVar)
+	watchNamespace, found := os.LookupEnv(k8sutil.WatchNamespaceEnvVar)
```

Comment:
We can use https://github.com/operator-framework/operator-sdk/blob/f30e17c5ed0d7c1942cac726fb567466cea61ca0/pkg/k8sutil/k8sutil.go#L33 here instead.

Reply:
We had been using that before, but I switched to this so that if the env variable is not set, we treat it the same as it being set to the empty string (i.e. `metav1.NamespaceAll`)

We could update the `GetWatchNamespace` function to behave that way, but that could be a breaking change for users.

Reply:
> We could update the GetWatchNamespace function to behave that way, but that could be a breaking change for users.

No, agreed we shouldn't break that it's not an internal pkg.